wait-for-statuses.py

```diff
@@ -46,10 +46,15 @@
       jobFailure = True
       break
 
+branch = subprocess.check_output("git rev-parse --abbrev-ref HEAD", text=True, shell=True)
```

Comment:
is there no gh actions variable for the branch name?

Reply:
There is GITHUB_REF. Updated code to make use of it